delivery_deliverea/__manifest__.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 FactorLibre - Jorge Martínez <jorge.martinez@factorlibre.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+{
+    "name": "Delivery Deliverea",
+    "summary": "Delivery Carrier implementation for Deliverea using their API",
+    "version": "16.0.1.0.0",
+    "category": "Stock",
+    "website": "https://github.com/OCA/delivery-carrier",
+    "author": "FactorLibre, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "base_delivery_carrier_label",
```

Comment:
We talked about not depending on this one, and for me is a totally blocker.

Reply:
You are right, I changed the code but I forgot to remove the dependency.